lib/client/chart.js

```diff
@@ -2,11 +2,8 @@
 
 var _ = require('lodash');
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['_' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1548085120/issues/source?bid=2452379&fileBranchId=3016446#l3)